frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/SlideInBackgroundable.java

```diff
@@ -77,14 +77,12 @@ public void run(ProgressIndicator indicator) {
         targetBranchLayout = branchLayout;
       } else {
         entryToSlideIn = entryByName.withChildren(List.empty());
-        targetBranchLayout = Try.of(() -> branchLayout.slideOut(slideInOptions.getName()))
-            .onFailure(EntryDoesNotExistException.class, exceptionWithMessageHandler(
-                format(getString("action.GitMachete.BaseSlideInBranchBelowAction.notification.message.entry-does-not-exist"),
-                    entryToSlideIn.getName())))
-            .onFailure(Exception.class, exceptionWithMessageHandler(/* message */ null))
-            .getOrNull();
-
-        if (targetBranchLayout == null) {
+        try {
```

Comment:
Actually... possibly the other use of `Try.of` (for `var newBranchLayout`) could be flattened back to `try ... catch`... and then `exceptionWithMessageHandler` can be just a `void` method instead of returning a `Consumer`